core/FamousEngine.js

```diff
@@ -387,7 +390,7 @@ FamousEngine.prototype.addScene = function addScene (scene) {
 
     var current = this._scenes[selector];
     if (current && current !== scene) current.dismount();
-    if (!scene.isMounted()) scene.mount(scene.getSelector());
+    if (!scene.isMounted()) scene.mount(scene._selector);
```

Comment:
@jd-carroll You are correct. Thanks. Seems like someone forgot to test after the rebase :rage4: (Note: that person was me).